package.json

```diff
@@ -80,22 +81,20 @@
     "lint:production:stylelint": "tsc --noCheck --outDir ./dist && stylelint '**/*.styles.ts' --custom-syntax postcss-lit --no-color",
     "test": "per-env",
     "test:development": "pnpm test:development:web-test-runner",
-    "test:development:playwright": "playwright test --quiet --ui-port 6007 --update-snapshots --update-source-method=overwrite",
-    "test:development:playwright:update": "rimraf ./src/aria-snapshots && playwright test --quiet --project aria --update-snapshots --update-source-method=overwrite --workers 100%",
-    "test:development:vitest": "vitest --config ./vitest.config.js & chokidar './src/eslint/**' './src/stylelint/**' --ignore './src/eslint/rules/*.test.ts' './src/stylelint/*.test.ts' --initial --silent --command 'tsc --noCheck --outDir ./dist'",
-    "test:development:vitest:comment": "Excluded from `test:development` because Vitest muddies the console when running alongside Web Test Runner.",
+    "test:development:playwright": "npm-run-all --parallel test:development:playwright:coverage test:development:playwright:ui",
+    "test:development:playwright:coverage": "http-server ./dist/playwright/coverage-report --port 6008 --silent",
```

Comment:
`http-server` is back! 

Playwright is able to serve coverage reports. But it would open a browser tab with the report every time this script is run. If you're okay with that, let me know. I'll replace `http-server` with the Playwright command.